schemas/README

```diff
@@ -30,6 +30,27 @@ Default values:
   - otherwise, there's no way to do client-side defaults, as the master would
     need to be involved in deciding all default values.
 
+  const.yaml:
```

Comment:
I either need to delete this file or update it, but what is written here is from before I started this PR, and it did not come true.